code/modules/projectiles/gun.dm

```diff
@@ -428,7 +428,9 @@ obj/item/weapon/gun/proc/newshot()
 
 	process_fire(target, user, 1, params)
 
-
+/obj/item/weapon/gun/proc/unlock() //used in summon guns and as a convience for admins
+	if(pin)	qdel(pin)
```

Comment:
people have been more anal about "code standards" lately.

I always liked single line if() changes all being on the same line, but whatever.